Dockerfile

```diff
@@ -9,20 +9,29 @@
 #
 FROM ubuntu:20.04
 
+WORKDIR /pwndbg
+
+ENV LANG en_US.utf8
 ENV TZ=America/New_York
 RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && \
     echo $TZ > /etc/timezone && \
     apt-get update && \
-    apt-get -y install vim golang
-
-ADD . /pwndbg/
+    apt-get install -y locales && \
+    rm -rf /var/lib/apt/lists/* && \
+    localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8 && \
+    apt-get update && \
+    apt-get install -y vim
 
-RUN cd /pwndbg && \
+ADD ./setup.sh /pwndbg/
+ADD ./requirements.txt /pwndbg/
+RUN sed -i "s/^git submodule/#git submodule/" ./setup.sh && \
```

Comment:
Not pass all the files, only the required files to do the set up. If we execute the `git submodule` here, we need all the code, in this step this command is commented, it will execute at the end of the Dockerfile.

Reply:
Can we add a comment above this line, so the purpose of commmenting out this in setup.sh is explicitly stated?

Anyway, great idea to suppress this here and re-add it later, so we don't waste time on rebuilding this each time :)

Reply:
Done in the commit c1be1e03305da11db2e4066ac25d4992a2d9b906